src/jobs/BenchmarkJob.jl

```diff
@@ -365,12 +365,15 @@ function execute_benchmarks!(job::BenchmarkJob, whichbuild::Symbol)
     run(`sudo cset set -d /user/child`)
     run(`sudo cset shield --reset`)
 
-    results = BenchmarkTools.load(benchresults, "results")
+    loaded = BenchmarkTools.load(benchresults)[2]
+    ind = findfirst(x->isa(x, BenchmarkTools.BenchmarkGroup), loaded)
+    @assert ind > 0
+    results = loaded[ind]
```

Comment:
Can you walk me through this? I had thought that `BenchmarkTools.load(benchresults)` would return `results::BenchmarkGroup` directly, since that what was saved via `BenchmarkTools.save` earlier. I'm probably misremembering something...

Reply:
With the JSON format, `save` writes
```
[{"Julia":"x.y.z","BenchmarkTools":"0.2.0"},[["BenchmarkTools.BenchmarkGroup",{"tags":[...],"data":{...}}]]]
```
Then `load` makes this
```
[Dict("Julia"=>"x.y.z", "BenchmarkTools"=>"0.2.0"), [BenchmarkGroup(...)]]
```
That is, the result is a two-element array where the first is the version information and the second is an array of the deserialized values.

```julia
loaded = BenchmarkTools.load(benchresults)[2] # gets the vector of values
ind = findfirst(x->isa(x, BenchmarkGroup), loaded) # figure out which of the elements is a BenchmarkGroup
@assert i > 0 # make sure one of them is
results = loaded[ind] # retrieve the BenchmarkGroup
```

Reply:
Oh. At some point in the future, we should change `load` to just return the second element, without the version info `Dict`. AFAICT there's no reason for `load` to expose version info to the caller. My bad, I should've caught this when reviewing JuliaCI/BenchmarkTools.jl#79. Let's ignore it for now, though, for poor Nanosoldier's sake.

Reply:
We'd might as well make that change now before too many packages start depending on the current behavior.

Reply:
Nah I'm just an idiot, this is already how it works.

Reply:
...well that was the easiest API patch ever 😛